addons/account/models/account_account.py

```diff
@@ -674,6 +674,11 @@ def _constraint_prefix_overlap(self):
         if res:
             raise ValidationError(_('Account Groups with the same granularity can\'t overlap'))
 
+    @api.constrains('parent_id')
+    def _check_parent_not_circular(self):
+        if not self._check_recursion():
+            raise ValidationError(_("Error ! You cannot create recursive group."))
```

Comment:
```suggestion
          raise ValidationError(_("You cannot create recursive group."))
```

Think it's better like this, it's already a ValidationError after all
![image](https://github.com/odoo/odoo/assets/1665365/5ac20411-49e2-4dfd-9583-983fab2ee073)

Reply:
I copy pasted it from other Validation error but seems ok to me.
![image](https://github.com/odoo/odoo/assets/47426643/50c8f28a-3249-40bb-9432-4db0df7afc7d)

 I'll make the change, thanks for your quick review !

Reply:
Some are like that, some aren't.
I think the ones without shout less and feel better.